core/php7.2Action/php.ini

```diff
@@ -0,0 +1,1948 @@
+; Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this file looks too big and boiler plate.
Can we have a short version with a few settings that we need to override

Reply:
Fixed.